index.html

```diff
@@ -1,3 +1,40 @@
-﻿
+<!doctype html>
+<html lang="en">
+<head>
+  <meta name="description" content="Demo project">
+  <title>The Best Chocolate Chip Cookies</title>
+</head>
+<body>
+  <h1>The Best Chocolate Chip Cookies</h1>
+    <section>
+      Recipe by Grandma
+      Prep Time: 45 Min
+    </section>
+    <h2>Ingredients</h2>
+      <ul style="list-style-type:circle;">
+        <li>1 1/2 cup (3 sticks) softened butter</li>
+        <li>1 cup brown sugar</li>
+        <li>1 cup granulated sugar</li>
+        <li>1 Tbl vanilla instant pudding powder</li>
+        <li>2 Tbl milk</li>
+        <li>2 Tbl vanilla extract</li>
+        <li>4 cups all purpose flour</li>
+        <li>2 tsp baking soda</li>
+        <li>4 cups chocolate chips</li>
+        <li>1 cup chopped walnuts or pecans(optional)</li>
+      </ul>
+      <h3>How to make Mama’s Recipe:</h3>
+        <section>
+          Preheat oven to 350 degrees Beat butter and sugars together until light and fluffy. Stir in pudding mix milk and vanilla extract. Beat in eggs. Add dry ingredients and stir until combined.
```

Comment:
This is a good section to try an ordered list `ol`. That will break the steps onto number lines.